adr/2024-11-2-assertion-format.md

```diff
@@ -0,0 +1,74 @@
+# [short title of solved problem and solution]
+
+* Status: proposed
+  <!-- will update below to only those who participated in the vote -->
+* Deciders: @gregsdennis @jdesrosiers @jviotti @mwadams @karenetheridge @awwright @benjam @relequestual
+* Date: 2024-11-02
+* Technical Story: https://github.com/json-schema-org/json-schema-spec/issues/1520
+* Voting issue: https://github.com/json-schema-org/TSC/issues/19
+
+## Context and Problem Statement
+
+There's a long and sticky history around format.
+
+1. Going back all the way to Draft 01, format has never required validation.
+2. Whether to support format validation has always been the decision of the implementation.
+3. The extent to which formats are validated has also been the decision of the implementation.
+
+The result of all of this is that implementation support for validation has been spotty at best. Despite the JSON Schema specs referencing very concretely defined formats (by referencing other specs), implementations that do support validation don't all support each format equally. This has been the primary driving force behind keeping format as an opt-in validation.
+
+With 2019-09, we decided that it was time to give the option of format validation to the schema author. They could enable validation by using a meta-schema which listed the Format Vocabulary with a true value, which meant, "format validation is required to process this schema."
+
+In 2020-12, we further refined this by offering two separate vocabularies, one that treats the keyword as an annotation and one that treats it as an assertion. The argument was that the behavior of a keyword shouldn't change based on whether the vocabulary was required or not.
+
+However, the fact remains that our users consistently report (via questions in Slack, GitHub, and StackOverflow) that they expect format to validate. (The most recent case I can think of was only last week, in .Net's effort to build a short-term solution for schema generation from types.)
+
+Due to this consistency in user expectations have decided to:
+
+1. make format an assertion keyword and strictly,
```

Comment:
> and strictly

This sentence feels awkward to me. I'm not sure it's clear what strictly means in this context. I know what you mean, but it can be stated more clearly for the sake of outside readers and maybe future us.

Reply:
I think this was a case where I was thinking of going somewhere, got distracted by something else, and didn't come back.  I have no idea what this was for now.  Will update.